### ISSUE TITLE
In the consumer mode Fstack has pool performance comparing to Linux socket

### ISSUE THREAD

**Issue by uvletter** [F-Stack/f-stack#774]
In the scene that a server only read data from client, the throughput of server based on Fstack is far less than Linux socket,
the example code likes the below:
```
if (event.filter == EVFILT_READ) {
            char buf[4096];
            ssize_t readlen = ff_read(clientfd, buf, sizeof(buf));
}
```
the client likes this:
```
        conn, err := net.Dial("tcp", addr)
	panicIfNeed(err)

	buf := make([]byte, sz)
	for {
		n, err := conn.Write(buf)
		panicIfNeed(err)

		if n != len(buf) {
			panic("short write")
		}
	}
```
In 4KB payload
| model | throughput(Byte/s) |
| --- | ---|
| Linux socket  | 1877778432 |
| Fstack | 672063488 |

If anyone has some ideas about it, thanks!

**Comment by CodingLappen** [comment]
Did you use two different machines? Do you actually use a SmartNIC or do run fstack in VM?

**Comment by uvletter** [comment]
nope, it's on the same physical machine. If you're interested, I would like to show the whole test code.

**Comment by jfb8856606** [comment]
The default configuration is optimized for large concurrency. If you only test a single connection, you need to modify some parameter configurations, such as `pkt_tx_delay`, etc.

And you're better off testing on two different servers, I don't know how you design your tests on a single service.

**Comment by jfb8856606** [comment]
As noted in the earlier comment, F-Stack's default configuration is optimized for high-concurrency workloads. A single-connection, one-way consumer benchmark is the worst case for F-Stack.

**Config changes for single-connection throughput:**
```ini
pkt_tx_delay=0
idle_sleep=0
net.inet.tcp.delayed_ack=0
```

**Additional notes:**
- Use multiple connections (`-c` in wrk) to utilize F-Stack's multi-core advantage
- Test on two separate machines to avoid loopback artifacts — F-Stack uses a physical DPDK NIC, not the Linux loopback interface, so same-machine testing introduces an unfair comparison path

See also: #758, #594, #664, #580
